gui-apps/kanshi/kanshi-1.1.0.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+DESCRIPTION="dynamic display configuration (autorandr for wayland)"
+HOMEPAGE="https://github.com/emersion/kanshi"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/emersion/kanshi"
+else
+	SRC_URI="https://github.com/emersion/kanshi/releases/download/v${PV}/${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm64 ~x86"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="doc"
+
+RDEPEND="
+	dev-libs/wayland
+"
+BDEPEND="
+	${RDEPEND}
+	doc? (
+		app-text/scdoc
+	)
+	virtual/pkgconfig
+	dev-libs/wayland-protocols
+"
+
+src_configure() {
+	local emesonargs=(
+		$(meson_feature doc man-pages)
```

Comment:
man-pages should be behind a USE flag.

Reply:
Also, doc should be behind a USE flag as well.

Reply:
nope
https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0301

Reply:
Ah, I'll make it always on then.

Reply:
We still want it to be configurable... Especially for folks working on things like embedded etc who strip lots of things.

Reply:
done